xml/System.Threading/AutoResetEvent.xml

```diff
@@ -53,7 +53,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>Represents a thread synchronization event that, when signaled, resets automatically after releasing a single waiting thread. This class cannot be inherited.</summary>
+    <summary>Represents a thread synchronization event that, when signaled, releases one single waiting thread, and the event resets automatically. If no thread is waiting, the next thread that is set to the waiting state is immediately released, and the event resets automatically. This class cannot be inherited.</summary>
```

Comment:
I don't know about the general guidelines, but I haven't found the summaries to be very useful in the past, I often find myself looking through the long and detailed remarks to find even basic info about how an API works.

Reply:
@stephentoub You are correct: https://github.com/dotnet/dotnet-api-docs/wiki/Summary and I should have noticed that. I'll send an updated PR.

Reply:
In that case maybe it can be made to be a bit more similar to the Windows docs for Create event:
- Parameter info: https://learn.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-createeventw#parameters
- Remarks: https://learn.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-createeventw#remarks

I think the main part to convey is that when a thread observes the signal on the event, the event auto-resets, such that it releases only one waiter, and that it remains signaled until a thread observes the signal. Anyway, it can probably be worded in multiple ways that work.